.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+##  Here is what I would like to see new or change within PokeNurse
+
+
```

Comment:
I think we should add a case here for if a user is reporting a bug. We should say something along the lines of `I would like to report a bug with PokeNurse, here are the reproduction steps`. What do you think?